HIP/hip-365.md

```diff
@@ -0,0 +1,638 @@
+---
+hip: 365
+title: Partially Fungible Tokens
+author: Albert Tam <albert.tam@hedera.com>
+working-group: John Shipman <jshipman@centre.io>, Chad Richman <crichman@centre.io>, Atul Mahamuni <atulmahamuni@hedera.com>, Mohsin Qamar <mohsin.qamar@hedera.com>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 2022-02-10
+discussions-to: 
```

Comment:
@superboo please add a new discussion topic in the Discussions tab and drop in a link here.